pkg/kubelet/server/server_test.go

```diff
@@ -1635,3 +1641,30 @@ func TestCRIHandler(t *testing.T) {
 	assert.Equal(t, path, fw.criHandler.RequestReceived.URL.Path)
 	assert.Equal(t, query, fw.criHandler.RequestReceived.URL.RawQuery)
 }
+
+func TestDebuggingDisabledHandlers(t *testing.T) {
+	fw := newServerTest(false)
+	defer fw.testHTTPServer.Close()
+
+	paths := []string{
+		"/run", "/exec", "/attach", "/portForward", "/containerLogs", "/runningpods",
+		"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/", "/runningpods/",
+		"/run/xxx", "/exec/xxx", "/attach/xxx",
```

Comment:
add `"/debug/pprof/profile"` and `"/logs/kubelet.log"`

Reply:
Done.